src/main/java/menion/android/whereyougo/geo/location/Location.java

```diff
@@ -7,10 +7,13 @@ public Location() {
     }
 
     public Location(android.location.Location loc) {
-        this(loc.getProvider(), loc.getLatitude(), loc.getLongitude());
+        this(loc.getLatitude(), loc.getLongitude());
         setTime(loc.getTime());
         if (loc.hasAccuracy()) {
-            setAccuracy(loc.getAccuracy());
+            setAccuracyHor(loc.getAccuracy());
+        }
+        if(loc.hasVerticalAccuracy()) {
+            setAccuracyVer(loc.getAccuracy());
```

Comment:
should be `            setAccuracyVer(loc.getVerticalAccuracyMeters());` ?

Maybe the implementation of `AccuracyVer` would be better in another PR.
There are some places where could be used, e.g. in `LocationState`.

Reply:
I agree, taken in 204f43dae38ae08ac69201ff99cea7d96cf5cfac.